### PR TITLE
Switch ExploreResponse type to new format

### PR DESCRIPTION
**Pull request by Andykaban**

### CODE REVIEW

File: response/types.go [hunk 1]
```diff
@@ -649,13 +649,291 @@ type SearchUserResponse struct {
...
+				ActorID     int    `json:"actor_id"`
+				SourceToken string `json:"source_token"`
+			} `json:"explore"`
+			ImpressionToken string `json:"impression_token"`
+		} `json:"media,omitempty"`
+	} `json:"items"`
 	NumResults          int    `json:"num_results"`
+	RankToken           string `json:"rank_token"`
 	AutoLoadMoreEnabled bool   `json:"auto_load_more_enabled"`
-	Items               []Item `json:"items"`
```
Comment (ahmdrz) [inline response/types.go]: I don't want to use inline structs in another structs.
So please update `Item` , Can you ?

Comment (Andykaban) [reply response/types.go]: Hello @ahmdrz . 
Sorry for late answer.
The Item struct in the ExploreResponse is incompatible with old the old used ItemStruct
May be better to make a separate structure ExploreItem  ?

Comment (ahmdrz) [reply response/types.go]: Are you sure there is one or more difference between Item and your ExploreItem struct ?
If you sure , Go for it and make ExploreItem.

Comment (Andykaban) [reply response/types.go]: Yes Item struct in the ExploreResponce has changed a lot. I dump a raw json document and save it in the my DropBox. 
https://www.dropbox.com/s/8wn9s68tenvku76/data.json?dl=0
You can download this file to see the changes
[data.json.txt](https://github.com/ahmdrz/goinsta/files/1657554/data.json.txt)

**Comment by ahmdrz** [comment]
Great , thank you ,  Make new struct for ExploreItem in this PR. And do not forget to creating new test functions , if needed.

On Jan 24, 2018 00:14, Andykaban <notifications@github.com> wrote:

@Andykaban commented on this pull request.

________________________________

In response/types.go<https://github.com/ahmdrz/goinsta/pull/92#discussion_r163371140>:

>       AutoLoadMoreEnabled bool   `json:"auto_load_more_enabled"`
-       Items               []Item `json:"items"`


Yes Item struct in the ExploreResponce has changed a lot. I dump a raw json document and save it in the my DropBox.
https://www.dropbox.com/s/8wn9s68tenvku76/data.json?dl=0
You can download this file to see the changes
data.json.txt<https://github.com/ahmdrz/goinsta/files/1657554/data.json.txt>

**Comment by Andykaban** [comment]
The Explore response changed again... I correct its structure and attach raw json
[json_new.txt](https://github.com/ahmdrz/goinsta/files/1661770/json_new.txt)

**Comment by ahmdrz** [comment]
Perfect , Thank you

**Pull request merged**